### PR TITLE
fix: issue-1168, remove duplicate CVE

### PR DESCRIPTION
**Pull request by michael12312**
Closes # 

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

**Comment by gyliu513** [comment]
Closed #1168

File: pkg/integration/trivy/analyzer.go [hunk 1]
```diff
@@ -49,16 +49,21 @@ func (TrivyAnalyzer) analyzeVulnerabilityReports(a common.Analyzer) ([]common.Re
 
 		// For each pod there may be multiple vulnerabilities
 		var failures []common.Failure
+		distinctFailures := make(map[string]common.Failure)
```
Comment (gyliu513) [inline pkg/integration/trivy/analyzer.go:52]: @michael12312 do you know why Trivy can report some duplicate CVEs for a specified pod? Is it a bug in Trivy?

Comment (michael12312) [reply pkg/integration/trivy/analyzer.go:52]: hi @gyliu513 

Per my testing, Trivy truely report duplicate CVEs. So for fixing, I made this PR.

Can you please help to do the code review and approval? Thanks very much!

**Comment by gyliu513** [comment]
it works for me, thanks @michael12312

@AlexsJones @matthisholleville ^^

**Review by AlexsJones — APPROVED** [review]

**Review by AlexsJones — APPROVED** [review]
An elegant fix, thanks

**Comment by michael12312** [comment]
hi @gyliu513 ,

@AlexsJones helped approved this PR, can you please help to merge this PR? and close the issue related?

**Pull request merged**